apps-rendering/src/components/Layout/NewsletterSignUpLayout.tsx

```diff
@@ -60,9 +104,38 @@ const NewsletterSignUpLayout: FC<Props> = ({ item, children }) => {
 					/>
 				</header>
 				<section css={contentRow}>
+					{!!newsletter && (
+						<div css={detailBlockStyles}>
+							<SvgNewsletter size="xsmall" />
+							<b>{newsletter.frequency}</b>
+							{/* TO DO - use regional focus, when on the MAPI type */}
+						</div>
+					)}
```

Comment:
we could use `maybeRender` here to help with this structure! `maybeRender` takes an `Option` as its first parameter, and then a function which returns React elements if that `Option` is a `some`:

```suggestion
					{maybeRender(item.promotedNewsletter, (newsletter) => (
						<div css={detailBlockStyles}>
							<SvgNewsletter size="xsmall" />
							<b>{newsletter.frequency}</b>
							{/* TO DO - use regional focus, when on the MAPI type */}
						</div>
					))}
```